### PR TITLE
app-misc/cdspeed, app-text/pep, media-sound/declick,... : EAPI7, improve ebuilds

### PR DESCRIPTION
**Pull request by mm1ke**
Hi,

This PR updates a few maintainer-needed ebuilds to EAPI7 (from EAPI2).
Please review.

### CODE REVIEW

**Comment by mm1ke** [comment]
@a17r 
Thx for the review, I've addressed all your points.

File: sys-fs/copyfs/copyfs-1.0.1-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+DESCRIPTION="fuse-based filesystem for maintaining configuration files"
+HOMEPAGE="https://boklm.eu/copyfs/"
+SRC_URI="https://boklm.eu/copyfs/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~x86 ~amd64 ~ppc"
+
+DEPEND="sys-apps/attr
+	>=sys-fs/fuse-2.0"
```
Comment (a17r) [inline sys-fs/copyfs/copyfs-1.0.1-r1.ebuild]: what about `RDEPEND`?

File: net-analyzer/nmbscan/nmbscan-1.2.5-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+DESCRIPTION="netbios scanner"
+HOMEPAGE="http://nmbscan.gbarbier.org/"
+SRC_URI="http://nmbscan.gbarbier.org/down/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86"
+
+RDEPEND="net-dns/bind-tools
```
Comment (a17r) [inline net-analyzer/nmbscan/nmbscan-1.2.5-r1.ebuild]: what about `DEPEND`?

File: media-sound/declick/declick-0.6.5-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="declick is a dynamic digital declicker for audio sample files"
```
Comment (a17r) [inline media-sound/declick/declick-0.6.5-r1.ebuild]: no `PN` in descriptions. Just start with `Dynamic digital...`

File: net-analyzer/nmbscan/nmbscan-1.2.5-r1.ebuild [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="netbios scanner"
```
Comment (a17r) [inline net-analyzer/nmbscan/nmbscan-1.2.5-r1.ebuild]: Maybe we should use the `NetBIOS` here, it seems to be the correct capitalisation.

File: sys-fs/copyfs/copyfs-1.0.1-r1.ebuild (hunk 1 above)
Comment (mm1ke) [reply sys-fs/copyfs/copyfs-1.0.1-r1.ebuild]: AFAICS, fuse should be in RDEPEND (as it's needed to work with the program), attr in DEPEND. Updated the ebuild.

File: net-analyzer/nmbscan/nmbscan-1.2.5-r1.ebuild (hunk 2 above)
Comment (mm1ke) [reply net-analyzer/nmbscan/nmbscan-1.2.5-r1.ebuild]: Actually, nmbscan is only a shell script, i think there is no need for a DEPEND?

File: media-sound/declick/declick-0.6.5-r1.ebuild (hunk 3 above)
Comment (mm1ke) [reply media-sound/declick/declick-0.6.5-r1.ebuild]: fixed :)

File: net-analyzer/nmbscan/nmbscan-1.2.5-r1.ebuild (hunk 4 above)
Comment (mm1ke) [reply net-analyzer/nmbscan/nmbscan-1.2.5-r1.ebuild]: I've updated the ebuild(s) and improved the description. While i was on it i've also fixed the SRC_URI and HOMEPAGE. (for the existing ebuild too)

File: net-analyzer/nmbscan/nmbscan-1.2.5-r1.ebuild (hunk 2 above)
Comment (a17r) [reply net-analyzer/nmbscan/nmbscan-1.2.5-r1.ebuild]: seems like, indeed, thanks for checking back.

**Comment by mm1ke** [comment]
@a17r 
I've removed the bug references, but don't know which bracket alignment need to be fixed. I've put the patch of nmbscan in a separate line but i don't see anything else which need to be updated.

**Comment by a17r** [comment]
@mm1ke closing bracket was indented, but don't worry about it ;)

**Pull request closed without merging**